search-app/src/cmr/search/services/query_service.clj

```diff
@@ -263,6 +265,21 @@
         results (qe/execute-query context query)]
     (common-search/search-results->response context query results)))
 
+(defn new-granules->collection-ids
```

Comment:
You should rename this. A `->` in a function name implies a transformation from one thing to another. So `new-granules->collection-ids` implies the primary argument should be a collection of granules, which it not. We tend to overuse `->` names in the CMR IMHO, but when we do use them one of the arguments is always of the type to the left of the `->` in the name.